### PR TITLE
feat: add withBorderOnScroll prop for Navigation component

### PR DESCRIPTION
**Pull request by goshander**

### CODE REVIEW

File: src/navigation/components/Navigation/Navigation.tsx [hunk 1]
```diff
@@ -43,6 +49,8 @@ export const Navigation: React.FC<NavigationProps> = ({data, logo, className}) =
     const onSidebarOpenedChange = (isOpen: boolean) => setIsSidebarOpened(isOpen);
 
     useEffect(() => {
+        if (!withBorderOnScroll) return () => {};
```
Comment (NikitaCG) [inline src/navigation/components/Navigation/Navigation.tsx:52]: I think we are no need new props to solve your problem from issue

File: src/navigation/components/Navigation/Navigation.tsx [hunk 2]
```diff
@@ -43,6 +49,8 @@ export const Navigation: React.FC<NavigationProps> = ({data, logo, className}) =
...
+
         const showBorderOnScroll = () => {
             if (!showBorder) {
```
Comment (NikitaCG) [inline src/navigation/components/Navigation/Navigation.tsx:55]: Change `!showBorder -> !withBorder` in this place, and if `window.scrollY === 0`, the border will disappear.
it is what you want ?)

**Review by NikitaCG — CHANGES_REQUESTED** [review]

File: src/navigation/components/Navigation/Navigation.tsx (hunk 1 above)
Comment (goshander) [reply src/navigation/components/Navigation/Navigation.tsx:52]: This PR is not related to the issue. Its two independents problems

Border no needed at all, like on this landing (css trick now)
https://ytsaurus.tech/

Comment (NikitaCG) [reply src/navigation/components/Navigation/Navigation.tsx:52]: ok

**Review by NikitaCG — APPROVED** [review]

**Pull request merged**